src/EnergyPlus/UnitarySystem.cc

```diff
@@ -2935,30 +2964,193 @@ namespace UnitarySystems {
                                                                    DataLoopNode::NodeConnectionType::Outlet,
                                                                    1,
                                                                    DataLoopNode::ObjectIsParent);
+        }
+
+        // need to read in all information needed to SetupOutputVariable in setupAllOutputVars
```

Comment:
This became really important over the last few days for setting up report variables.